packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -536,17 +536,13 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
   }
 
   private _updatePages(props?: IListProps) {
-    let { items, startIndex, renderCount } = (props || this.props);
-
-    renderCount = this._getRenderCount(props);
-
     // console.log('updating pages');
 
     if (!this._requiredRect) {
       this._updateRenderRects(props);
     }
 
-    let newListState = this._buildPages(items as any[], startIndex as number, renderCount);
+    let newListState = this._buildPages(props);
```

Comment:
isn't props marked as optional for `_updatePages` but required for `_buildPages`?

Reply:
it should be defaulting to this.props